lib/asciidoctor-pdf/theme_loader.rb

```diff
@@ -86,8 +86,10 @@ def load hash, theme_data = nil
   private
 
   def process_entry key, val, data
-    if key == "admonition_icons"
-      data[key] = symbolize_keys val
+    if key.start_with? 'admonition_icon_'
```

Comment:
Correct. I realized this would be more consistent with the other keys and read better as `@theme.admonition_icon_tip`.